synapse/federation/sender/per_destination_queue.py

```diff
@@ -422,16 +421,12 @@ async def _transaction_transmission_loop(self) -> None:
                 logger.info(
                     "Failed to send event %s to %s", p.event_id, self._destination
                 )
-
-            self._start_catching_up()
         except Exception:
             logger.exception("TX [%s] Failed to send transaction", self._destination)
             for p in pending_pdus:
                 logger.info(
                     "Failed to send event %s to %s", p.event_id, self._destination
                 )
```

Comment:
Unrelated, but I wonder if we should try to log the event IDs on one line here instead of 50. In another PR though.